dandi/dandiapi.py

```diff
@@ -729,13 +847,63 @@ def api_path(self) -> str:
     def download_url(self) -> str:
         return self.client.get_url(f"{self.api_path}download/")
 
+    def get_metadata(self) -> models.Asset:
+        """
+        Fetch the metadata for the asset as a `dandischema.models.Asset`
+        instance
+        """
+        return models.Asset.parse_obj(self.get_raw_metadata())
+
     def get_raw_metadata(self) -> Dict[str, Any]:
         """Fetch the metadata for the asset as an unprocessed `dict`"""
         if self._metadata is not None:
             return self._metadata
         else:
             return cast(Dict[str, Any], self.client.get(self.api_path))
 
+    def get_digest(self, digest_type: Union[str, models.DigestType]) -> str:
```

Comment:
Let's default to our `dandi-etag` may be?

Reply:
Done.